capa/features/extractors/smda/file.py

```diff
@@ -67,7 +67,8 @@ def extract_file_export_names(smda_report, file_path):
     lief_binary = lief.parse(file_path)
     if lief_binary is not None:
         for function in lief_binary.exported_functions:
-            yield Export(function.name), function.address
+            if function and not isinstance(function, str):
```

Comment:
in what circumstances does lief return something invalid here? is it for ordinals or something that we might care about?

Reply:
im trying to spot that, but also apart of string i saw `None` is why i put function to ensure that is `True`

Reply:
ok found the problem, is lief version, i had 
```
>>> lief.__version__
'0.9.0-a448c5e'
```


with 
```
>>> lief.__version__
'0.10.1-bfe5414'
```

it works just fine, maybe it would be good to ensure that version >9 is installed?

Reply:
yeah, lets fix this by pinning the lief version

Reply:
this dep comes transitively from smda, so perhaps we should suggest @jcrussell pin the dependency there?

Reply:
Seems like a good idea, did you test with 0.10.0?

@danielplohmann

Reply:
Hey!
Confirmed with LIEF 0.10.1. 
I now pinned the Python3 dependency LIEF to this exact version.

Reply:
cool, thank you Daniel, i will close this RP then